README.html

```diff
@@ -161,6 +162,44 @@ <h3>Initial Client Connection</h3>
 <p>From this point on, the calls below can be done to the local_addr or the relay_addr.</p>
 <p>If the client is in local mode, the API endpoint is the address given by <code>local_addr</code>. This should be the preferred mode if it's available.</p>
 <p>If the client is in remote mode, the API endpoint is the address given by <code>relay_addr</code>.</p>
+<h3>HDA User Authentication</h3>
+<p>Since different users can have different access permissions on HDA so each user must identify himself/herself to the HDA. In order to do that the app must request a PIN from the user. This PIN can be sent to the HDA either directly or via Proxy.</p>
+<p>To authenticate, the client app has to issue a <code>POST</code> to <code>/auth</code> with an <code>application/json</code> content-type body containing the following in it:</p>
+<ul>
+<li>
+<p><code>POST /auth</code></p>
+<pre lang="json"><code>  {
+    &quot;pin&quot;: &quot;1234&quot;
+  }
+</code></pre>
+<ul>
+<li>The pin must be string and not numeric.</li>
```

Comment:
`string` is a superset of `numeric` so this is redundant. 

i rather if we state that the PIN must be 3 to 5 characters ant it must match `[A-Za-z0-9]+`